glossary.md

```diff
@@ -30,9 +30,7 @@ It reads the [configuration files](#configuration) from a [bundle](#bundle), use
 
 ## <a name="glossaryRuntimeNamespace" />Runtime namespace
 
-On Linux, a leaf in the [namespace][namespaces.7] hierarchy from which the [runtime](#runtime) process is executed.
-New container namespaces will be created as children of the runtime namespaces.
-
+On Linux, the namespaces from which new [container namespaces](config-linux.md#namespaces) are created.
```

Comment:
Why not link to the glossary container namespace above? Looks more suitable to me.